src/main/scala/uk/gov/homeoffice/drt/service/ArrivalService.scala

```diff
@@ -0,0 +1,33 @@
+package uk.gov.homeoffice.drt.service
+
+import cats.effect.Sync
+import cats.syntax.all._
+import org.slf4j.LoggerFactory
+import uk.gov.homeoffice.drt.{ResourceObject, model}
+import uk.gov.homeoffice.drt.model.{Arrival, DepartureAirport, RequestedFlightDetails}
+import uk.gov.homeoffice.drt.repository.{ArrivalRepositoryI, ArrivalTableData}
+import uk.gov.homeoffice.drt.utils.DateUtil._
+import ResourceObject.{getClass, _}
+class ArrivalService[F[_] : Sync](arrivalsRepository: ArrivalRepositoryI[F]) {
```

Comment:
Add some white space between the imports and the class def - or just use intellij's auto-format shortcut to do this for you